frontend/src/app/modules/user/user-info-editor/user-info-editor.component.html

```diff
@@ -57,6 +57,17 @@
                     </div>
                 </div>
             </div>
+            <p>ACCOUNTS</p>
```

Comment:
could you please create a separate block (in screen we should have 2 gray blocks in black background)

Reply:
is that okay?

![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/56970649/4761d7b7-f673-4347-9552-eacef9efed8c)

![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/56970649/ae5eb318-5cbf-4531-8820-f5d93274b66e)